.meteor/packages

```diff
@@ -27,9 +27,11 @@ sacha:spin
 multiply:iron-router-progress
 zeroasterisk:throttle
 percolate:synced-cron
-accounts-facebook
 service-configuration
 yang2007chun:materialize-scss
 digilord:faker
+meteorhacks:kadira-debug
```

Comment:
I had removed the package and now I have committed and pushed but I still see the kadira lines here.  Is there more I have to do to remove kadira yet?